docs/src/tutorials/Add-New-Network.md

```diff
@@ -0,0 +1,163 @@
+# Add New Network
+
+Parity Signer supports adding a Substrate-based network or update the existing network via QR code.
+
+After you've installed [required software](#Prerequisites), you need to add *Network's Spec* to Signer and add *Network Metadata* for this network, so that Signer is able to decode, and you could read and verify transactions you are signing on this network.
+
+If you are updating metadata for already existing network you only need to update *Network Metadata*.
+
+s*Network Specs*
+
+1. [Get](#get-network-spec)
+2. [Sign](#sign-network-spec)
+3. [Feed into Signer](#feed-network-spec-into-signer)
+
+*Network Metadata*
+
+4. [Get](#get-network-metadata)
+5. [Sign](#sign-network-metadata)
+6. [Feed into Signer](#feed-network-metadata-into-signer)
+
+### Prerequisites
+
+- [ ] Websocket URL of the network (we maintain an [address book](https://github.com/paritytech/parity-signer/blob/master/rust/generate_message/address_book))
+- [ ] [rust](https://www.rust-lang.org/tools/install)
+- [ ] [subkey](https://docs.substrate.io/v3/tools/subkey/#installation)
+- [ ] a clone of [parity-signer repository](https://github.com/paritytech/parity-signer). In the repository go to `generate_message` crate:
+
+```
+cd rust/generate_message
+cargo build
+cargo run restore_defaults
+```
+
+With `subkey` you need to generate a private and public keys (`<secret-phrase>` and `Public key (hex)`). You will be able to update a network only with metadata that is signed with the same keys as network specs that defined the network for your Signer instance.
+
+```
+subkey generate
+```
+
+---
+
+## Network Specs
+
+### Get Network Specs
+
+In `parity-signer/rust/generate_message`
+
+```
+cargo run add_spec -u <network-ws-url> -<crypto>
```

Comment:
Not sure if we should suggest for users to maintain or to not maintain the hot database.
If the database is not updated, as with the metadata update in line 122 below, `-d` key also should be added, i.e.:
`cargo run add_spec -d -u <network-ws-url> -<crypto>`
If the network is added in the database, it could be referred to by name as well. This could be both useful and dangerous.

Reply:
Added "Hint: you can read more about [interface with hot database](https://github.com/paritytech/parity-signer/blob/master/rust/generate_message/src/lib.rs) if you want to maintain it"